app/views/recipes/show.html.haml

```diff
@@ -0,0 +1,24 @@
+.page-header
+  %h1.text-uppercase.text-center Recipes Details
+
+.row
+  .col-md-4.col-md-offset-4
+    %p
+      %strong Name: 
+      #{@recipe.name}  
+    %p
+      %strong Preparation:
+      %br
+      #{@recipe.preparation}
+    %p
+      %strong Created On: 
+      #{@recipe.created_at.strftime('%d %B %Y')}
```

Comment:
No, you just have to add the format to the `en.yml` file and use it from the view.